.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,14 @@
+{
+    "image": "netdata/devenv", 
+    "extensions":[
+		"golang.go",
```

Comment:
Only this - if i want to develop in python i get goland extensions installed.

If that is ok and it is your plan i approve it

Reply:
The idea is to have a devenv for the whole suit of Netdata. I think it's more flexible and the overhead is truly minimum. All the extensions take a couple of seconds to be installed. The dependencies already exist inside the container since we have 1 container for both netdata, go.d and dashboard.

Reply:
So that's OK from me :P